sql_runner/snowflake_target.go

```diff
@@ -30,6 +31,7 @@ func (sft SnowFlakeTarget) IsConnectable() bool {
 }
 
 func NewSnowflakeTarget(target Target) *SnowFlakeTarget {
+	// Note: region connection parameter is deprecated
```

Comment:
// Note: region connection parameter is deprecated


Does it still work? What is to be used in place of it?

Reply:
It still works as it is. 
The suggestion for the region connection parameter is to be included in `:account:` (https://godoc.org/github.com/snowflakedb/gosnowflake#hdr-Connection_Parameters). So it has to do with the target configuration in playbooks.